dev/integration_tests/ios_host_app/FlutterUITests/FlutterUITests.m

```diff
@@ -3,6 +3,9 @@
 // found in the LICENSE file.
 
 @import XCTest;
+@import OSLog;
+
+static const CGFloat kStandardTimeout = 120.0;
```

Comment:
Wow 2 minutes?  Doesn't that sound excessive?

Reply:
The CI was hitting 60 second timeouts, I wanted to make it much longer to see what happens.(I think 60 s timeout is long enough too. The issue is probably because the app hangs. )